### DISCUSSION TITLE
Quick Start Guide | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1792] [Q&A]
# Quick Start Guide | Seeed Studio Wiki

SenseCAP Watcher is an AI watcher to help you monitor anomalies within a space and then take actions. Simply speak to Watcher to give commands, and it will execute your desired tasks when the event occurs.

https://wiki.seeedstudio.com/getting_started_with_watcher/

**Comment by baqwas** [comment]
Hello! Is there a "desktop" app (for Linux) that can be used in lieu of the mobile app?

Xie xie.

**Comment by baqwas** [comment]
As of last year, SquareLine Studio did not support mp4 files. Perhaps, some future extension in the Watcher will permit playback of video files?

**Comment by mafifkasno** [comment]
I already bought recamera gimbal. its so good for me. Currently I want the Yolo11 model to detect person face. Please teach me on how to modify the current Yolo11 model to detect face. Thank you.

**Reply by nfs0619** [reply]
Hi, thank you for your support about the reCamera Gimbal!

To modify the current YOLO11 model to detect faces, you can refer to our model conversion guide here:
https://wiki.seeedstudio.com/recamera\_model\_conversion/

If you run into any issues while following the guide, feel free to reach out to our technical support team at **[techsupport@seeed.io](mailto:techsupport@seeed.io)** — they’ll be able to provide more detailed and timely assistance.

**Comment by eerke2021** [comment]
The QR code for the ReCamera 2002 HQ PoE 64GB comes here. It isn’t listed in the types of camera above. Has it got its own page or is it no different than above?
